bin/database.py

```diff
@@ -1067,21 +1068,21 @@ def upgrade_to_30_closure_analysis(coll_item, coll):
                 if 'created' not in file_:
                     file_['created'] = analysis_.get('created', datetime.datetime(1970, 1, 1))
         result = config.db[coll].update_one({'_id': coll_item['_id']}, {'$set': {'analyses': analyses}})
-        if result.modified_count == 1:
+        if result.modified_count == 1 or files == []:
```

Comment:
Specifically, Ryan is seeing this error on dev which is probably caused by an empty list of analyses:
```
Traceback (most recent call last):
  File "/var/scitran/code/api/bin/database.py", line 1146, in upgrade_schema
    globals()[upgrade_script]()
  File "/var/scitran/code/api/bin/database.py", line 1101, in upgrade_to_30
    process_cursor(cursor, upgrade_to_30_closure_analysis, context = 'sessions')
  File "/var/scitran/code/api/bin/database.py", line 117, in process_cursor
    result = closure(document, context)
  File "/var/scitran/code/api/bin/database.py", line 1071, in upgrade_to_30_closure_analysis
    if result.modified_count == 1 or files == []:
UnboundLocalError: local variable 'files' referenced before assignment
```